### PR TITLE
[FR] Update `MWC 2014`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review) > @DoryanR I edited again so you will need to review the new commit 2ac4b859bb354359260dae434c32df4fc749622f

### CODE REVIEW

File: wiki/Tournaments/MWC/2014/fr.md [hunk 1]
```diff
@@ -484,80 +484,80 @@ Cette compétition s'est conclue sur le podium suivant :
 | **Corée du Sud** ::{ flag=KR }:: | **4** | 0 | ::{ flag=GB }:: Royaume-Uni | [#1](https://osu.ppy.sh/community/matches/8587506) |
 | **Australie** ::{ flag=AU }:: | **4** | 1 | ::{ flag=DE }:: Allemagne | [#1](https://osu.ppy.sh/community/matches/8587531) |
 
-## Réglement
+## Règlement
 
 ### Règles du tournoi
 
-1. L'osu!mania World cup est un tournoi, séparé en 4-keys et 7-keys avec une équipe par pays.
+1. L'osu!mania World cup est un tournoi par pays, séparé en 4-keys et 7-keys avec une équipe.
```
Comment (DoryanR) [inline wiki/Tournaments/MWC/2014/fr.md:491]:
```suggestion
1. L'osu!mania World Cup est un tournoi par pays, séparé en 4-keys et 7-keys avec une équipe.
```

File: wiki/Tournaments/MWC/2014/fr.md [hunk 2]
```diff
@@ -484,80 +484,80 @@ Cette compétition s'est conclue sur le podium suivant :
...
 2. Les beatmaps prévues pour chaque phase seront annoncées publiquement par l'équipe de sélection des beatmaps le dimanche précédent le début de chaque phase. Aucune autre beatmap ne peut être jouée durant les matchs.
    - Une beatmap, le tiebreaker, servira à départager deux équipes à égalité à la fin d'un match. Elle ne peut être utilisée pour une autre fin.
-3. Les dates et horaires des matchs seront fixés par la direction (cf. rubrique « Planification des matchs »).
-4. Si aucun membre de la direction ou aucun arbitre n'est présent à l'heure du match, celui-ci sera reporté à une date ultérieure.
+3. Les dates et horaires des matchs seront fixées par la direction (cf. rubrique « Planification des matchs »).
+4. Si aucun membre de la direction ou aucun arbitre n'est présent à l'heure du match, celui-ci sera reporté.
 5. En cas de Fail le score ne sera pas comptabilisé.
-   - Les revives (revenir d'un statut d'échec à un statut normal au sein d'une partie en remplissant la barre de vie au maximum) sont autorisés.
-6. L'utilisation des Visual Settings est autorisée.
+   - Revivre et de survivre pendant une carte est considéré comme une réussite.
```
Comment (DoryanR) [inline wiki/Tournaments/MWC/2014/fr.md:497]:
```suggestion
   - Le fait de revivre et de survivre au cours d'une map est considéré comme une réussite.
```

File: wiki/Tournaments/MWC/2014/fr.md [hunk 3]
```diff
@@ -484,80 +484,80 @@ Cette compétition s'est conclue sur le podium suivant :
...
-   1. La taille d'une équipe ne peut dépasser 6 joueurs.
-   2. Le responsable d'une équipe, le capitaine, doit être spécifié.
-   3. Vous pouvez enregistrer votre équipe pour les deux tournois si vous voulez. Vous devez pour cela recourir à la phase d'enregistrement deux fois.
-      1. Si vous vous enregistrer pour les deux tournois les deux équipes n'ont pas besoin d'être identiques.
-      2. Si votre pays s'enregistre pour le tournoi 4K elle n'est pas obligée de s'inscrire aussi pour le tournoi 7K et vice-versa.
+1. Chaque équipe doit être constituée **au minimum de 2 joueurs**.
+   - La taille d'une équipe ne peut dépasser 4 joueurs.
+   - Le responsable d'une équipe, le capitaine, doit être spécifié.
+   - Vous pouvez enregistrer votre équipe pour les deux tournois si vous voulez. Vous devez pour cela recourir à la phase d'enregistrement deux fois.
+     - Si vous vous enregistrez pour les deux tournois, les deux équipes n'ont pas besoin d'être identiques.
```
Comment (DoryanR) [inline wiki/Tournaments/MWC/2014/fr.md:516]:
```suggestion
     - Si vous êtes enregistrez pour les deux tournois, les deux équipes n'ont pas besoin d'être identiques.
```

File: wiki/Tournaments/MWC/2014/fr.md [hunk 4]
```diff
@@ -484,80 +484,80 @@ Cette compétition s'est conclue sur le podium suivant :
...
-5. Les matchs suivant sont tous à élimination directe. Cela signifique que l'équipe victorieuse passe au tour suivant alors que l'autre équipes est éliminée du tournoi.
-6. **Conditions de victoire :**
-   - Durant la phase de Poules, Vous devez gagner 4 beatmaps pour gagner un match. (Meilleur des 7)
-   - Durant les huitièmes et quart de finale, vous devez gagner 5 beatmaps pour gagner un match. (Meilleur des 9)
-   - Durant les demi-finales et finale, vous devez gagner 6 beatmaps pour gagner un match. (Meilleur des 11)
+5. Les matchs suivants sont tous à élimination directe. Cela signifie que l'équipe victorieuse passe au tour suivant, alors que les autres équipes sont éliminées du tournoi.
+6. Les **conditions de victoire** pour chaque étape seront :
+   - Durant les phases de poules, vous devez gagner 4 beatmaps pour gagner un match. (Meilleur des 7)
+   - Durant les huitièmes et quarts de finale, vous devez gagner 5 beatmaps pour gagner un match. (Meilleur des 9)
+   - Durant les demi-finales et finales, vous devez gagner 6 beatmaps pour gagner un match. (Meilleur dès 11)
```
Comment (DoryanR) [inline wiki/Tournaments/MWC/2014/fr.md:544]:
```suggestion
   - Durant les demi-finales et finales, vous devez gagner 6 beatmaps pour gagner un match. (Meilleur des 11)
```

**Review by DoryanR — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/MWC/2014/fr.md (hunk 3 above)
Comment (agatemosu) [reply wiki/Tournaments/MWC/2014/fr.md:516]: vous vous

**Review by DoryanR — APPROVED** [review]

**Pull request merged**